src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,50 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private double side;
+    private double firstBases;
+    private double secondBases;
```

Comment:
Name of variable in multiple form is suitable for arrays, lists and other collections but for single value better to use singular
```suggestion
    private double firstBase;
    private double secondBase;
```